### ISSUE TITLE
Problem with the Maneuver Planer

### ISSUE THREAD

**Issue by firesoft-de** [MuMech/MechJeb2#1000]
I'm using MechJeb 2.7 (via CKAN) with a bunch of other mods ([mod_list.txt](https://github.com/MuMech/MechJeb2/files/1785514/mod_list.txt)) and the current KSP steam version.
Since the last KSP update I'm encountering some problems with the Maneuver Planer.

For example I am launching a supply vessel with the ascent guidance to an 80km Orbit with the planes matched to my space station. Then I select the planer and create a hohmann transfer. When I select execute the waypoint is deleted instead of executed. The same applies for every retry.

At this point I thought "Lets do it manually". So i just created the waypoint (with the planer), used the warp helper to jump 2min before the execution point.
At the end of the warp process I was unable to do any maneuvering. The indicators at the left bottom are just wobbling around (it looks like the something is trying to lock on a non existing node). But I am still able to set engine thrust.

I think that this problem is linked to the maneuver problem. In my mind the execution process is started and then crash. Because of this the waypoint is deleted. But after the time warp, parts of the execution process still try to turn and hold the vessel in the direction needed for the failed maneuver.

I am experiencing the problems with all my crafts.

So long story short:
1. the maneuver planer is not working for me (for me it sounds like #591)
2. lost all controls after time warp

If you need any further files just tell me.

**Comment by firesoft-de** [comment]
Error 2 is releated to RemoteTech. It was a wrong setting in the Flight Computer (upper left corner).

**Comment by sarbian** [comment]
I would love to see the log because the "the waypoint is deleted instead of executed" reeks of an exception.

**Comment by firesoft-de** [comment]
I attached the log of my last flight. The problem happend just once, short before landing (or better say blowing a crater into mun :/ ).
If I remember correctly it should be somewhere after 12:00.

The only thing I noticed in the is a RemoteTech Error at [11:53:03.951] and a bunch "material from string" Errors

[KSP.log](https://github.com/MuMech/MechJeb2/files/1788911/KSP.log)

**Comment by firesoft-de** [comment]
I'm having the error right now. Unfortunately there ist nothing in the log. I attached the savegame, the CKAN mod list ([mechjeb_error.zip](https://github.com/MuMech/MechJeb2/files/1790433/mechjeb_error.zip)) and the log ([KSP.log](https://github.com/MuMech/MechJeb2/files/1790436/KSP.log)).

## Situation
### Craft 
MUNEXP 22

### Position
Orbit around Mun

### Intention
I want to change the periapsis and the apoapsis. Therefore I first use the Maneuver Planner to set up the Apoapsis node (100km).  Then I set the Periapsis Node (99.9km). Then I start execution by "Execute all nodes". 

The first node is deleted. The second node is still there and Mechjeb starts to warp towards it.

Also when trying to execute a manually set node it is deleted instead of executed
